resources/public/api/conf/2.0/application.raml

```diff
@@ -422,7 +423,7 @@ traits:
 
         This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
 
-        A successful response will contain a URI and an arrival ID which you can use to retrieve messages related to the movement.
+        A successful response will contain a URI which you can use to retrieve the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.  
 
```

Comment:
Same issue as above.

Suggestion:
A successful response will contain a URI that you can use to get the message status.